tests/e2e/chaos/chainlink-relay-sol.yaml

```diff
@@ -9,14 +9,15 @@ charts:
   chainlink:
     index: 4
     values:
-      replicas: 19
+      replicas: 5
```

Comment:
why reduce this down to 5? ideally we would like to test with what we use on mainnet (16 oracles + 1 bootstrap)

Reply:
It's a local run template, CI is still running 19 nodes, but we are planning to shrink it soon.